docs/banking.ipynb

```diff
@@ -8,6 +8,8 @@
         "\n",
         "This notebook demonstrates how to use the `pg_diffix`, a PostgreSQL extention that implements Diffix Elm anonymization.\n",
         "\n",
+        "> Diffix Elm used here is missing several features of the next version Fir being work-in-progress in the `master` branch.\n",
```

Comment:
This is the `master` branch and won't appear in other branches, so no point in mentioning it.
Maybe limit the sentence to:
"Diffix Elm used here is missing several features of the next version Fir, which is still a work in progress."